codes/classical/q-ary_digits/skew_cyclic.yml

```diff
@@ -0,0 +1,119 @@
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Unknown.'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  # encoders: 
+  #   - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+  #   - 'Measurement-based preparation ... with ancilla overhead of ... .'
+  #   - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+  #   - 'Universal gate set achieved by either additional ... gate.'
+  #   - 'Magic-state distillation protocols'
+  #   - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 
+    - 'Only given for Skew-BCH codes, adapted froom standard BCH codes.'
+    # - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  threshold: 
+    - 'Unknown'
+  
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'Not directly implemented, but BCH codes form a subclass, and are used in DVD, solid state drive storage, etc.'
+
+notes:
+  - 'Computer algebra software is used to find most codes of this type. \cite{arXiv:math/0604603} give several examples of codes, which have slightly improved minimum distance for some \((n,k)\) codes.'
+  - '\cite{doi:10.1016/j.disc.2020.112189} constructs quantum CSS codes from classical skew-cylic codes.'
+  # - 'Bounds on n,k, or d for this class.'
+  # - 'Links to code tables, github, GAP algebra packages, more papers \cite{arXiv:####.#####}.'
+  # - 'Connections to physics; based on \(H^4(\mathcal{BG},\mathbb{Z})\)'
+  # - 'Can reference other codes, but try to put such connections in the relations fields.'
+  # - 'Anything noteworthy that''s not above or below.'
+
+relations:
+  parents:
+    - code_id: q-ary_linear
+    #   detail: 'Proven \cite{doi:10.1504/IJICOT.2011.044674}'
+    # - code_id: code_id2
+    #   detail: 'code_id2 is a code family that is constructed in a seemingly different way than this code, yet still includes this code.'
+    # - code_id: code_id3
+    #   detail: 'code_id3 is code family described by a property shared by this code other than physical space (e.g., topological)'
+
+  cousins:
+    - code_id: quasi_cyclic
```

Comment:
Only one cousin connection is needed, so you need to include both codes in the description. E.g., "Quasi-cylic and skew-cyclic codes are equivalent \cite{}." However, is it true that any quasi-cyclic code is also skew-cyclic? If so, then I would argue we should combine the two entries into one because having two identical code families seems confusing.